Compiling on Windows fails

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

Run build of server and tools on Windows

### 2. What did you expect to see? (Required)

Successful build

### 3. What did you see instead (Required)

```
 # github.com/pingcap/tidb/ddl/ingest
Error: ddl\ingest\env.go:66:29: too many arguments in call to util.GenRLimit
	have (string)
	want ()
```

```
# github.com/pingcap/tidb/tidb-server
tidb-server/main.go:249:28: cannot use func() {…} (value of type func()) as func(bool) value in argument to signal.SetupSignalHandler
```

```
# github.com/pingcap/tidb/br/pkg/lightning/importer
br/pkg/lightning/importer/import.go:335:20: undefined: local.RlimT
br/pkg/lightning/importer/import.go:340:38: undefined: local.RlimT
```

### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->

--- a/br/pkg/lightning/backend/local/local_windows.go
+++ b/br/pkg/lightning/backend/local/local_windows.go
@@ -22,7 +22,8 @@ import (
 	"github.com/pingcap/errors"
 )
 
-type Rlim_t = uint64
+// RlimT is the type of rlimit values.
+type RlimT = uint64
 
 // return a big value as unlimited, since rlimit verify is skipped in windows.
 func GetSystemRLimit() (uint64, error) {

--- a/util/rlimit_windows.go
+++ b/util/rlimit_windows.go
@@ -16,6 +16,6 @@
 package util
 
 // GenRLimit always return 1024.
-func GenRLimit() uint64 {
+func GenRLimit(source string) uint64 {
 	return 1024
 }

--- a/util/signal/signal_windows.go
+++ b/util/signal/signal_windows.go
@@ -25,7 +25,7 @@ import (
 )
 
 // SetupSignalHandler setup signal handler for TiDB Server
-func SetupSignalHandler(shutdownFunc func(bool)) {
+func SetupSignalHandler(shutdownFunc func()) {
 	//todo deal with dump goroutine stack on windows
 	closeSignalChan := make(chan os.Signal, 1)
 	signal.Notify(closeSignalChan,
@@ -37,6 +37,6 @@ func SetupSignalHandler(shutdownFunc func(bool)) {
 	go func() {
 		sig := <-closeSignalChan
 		logutil.BgLogger().Info("got signal to exit", zap.Stringer("signal", sig))
-		shutdownFunc(sig != syscall.SIGHUP)
+		shutdownFunc()
 	}()
 }

